tests/lib/test_user.py

```
@@ -24,3 +49,104 @@ def test_even_if_token_set_for_user_zero_authenticate_still_fails():
+
+    encrypted_data = encrypt_data(data)
+
+    assert encrypted_data.startswith(ENCRYPTED_VALUE_PREFIX)
+    assert encrypted_data != data
+    assert encrypted_data.startswith(ENCRYPTED_VALUE_PREFIX) is True
+    assert decrypt_data(encrypted_data) == data
+
+def test_encrypt_data_returns_data_without_public_key(tmp_path):
+    try:
+        _override_security_config(tmp_path, 'security: {}\n')
+        assert encrypt_data('secret value') is 'secret value'
```

Pylint (4.0.5)

[refactor] In 'encrypt_data('secret value') is 'secret value'', use '==' when comparing constant literals not 'is' ('encrypt_data('secret value') == 'secret value'')

(R0123)